### ISSUE TITLE
A potential bug of NPD related to usage of libogg

### ISSUE THREAD

**Issue by ash1852** [xiph/opusfile#37]
Hi, I found a potential null pointer dereference bug in the project source code of opusfile related to usage of libogg, and I have shown the execution sequence of the program that may generate the bug on the graph below. The red text illustrates the steps that generate the bug, the red arrows represent the control flow.
![os_lacing_expand](https://user-images.githubusercontent.com/87304478/189794715-c62e372f-4a84-45e5-bef3-c9175aa51558.jpg)
in short, the key step is not check the return value of call-statement relate to libogg.
would you help to check if this bug is true?thank you for your patience and effort.
